osu.Game.Tests/Editing/Checks/CheckDrainTimeTest.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Edit.Checks;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Tests.Beatmaps;
+
+namespace osu.Game.Tests.Editing.Checks
+{
+    public class CheckDrainTimeTest
+    {
+        private CheckDrainTime check = null!;
+
+        private IBeatmap beatmap = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            check = new CheckDrainTime();
+        }
+
+        [Test]
+        public void TestDrainTimeShort()
+        {
+            setShortDrainTimeBeatmap();
+            var content = new BeatmapVerifierContext(beatmap, new TestWorkingBeatmap(beatmap));
```

Comment:
So `setShortDrainTimeBeatmap()` mutates a field of the test, which then is passed to `BeatmapVerifierContext` via the field. This muddles cause and effect when reading this code.

Please change `void setShortDrainTimeBeatmap()` to `IBeatmap getShortDrainTimeBeatmap()` and pass that to `BeatmapVerifierContext` explicitly.

Reply:
This was what I initially had, but I changed it after comparing to https://github.com/cdwcgt/osu/blob/master/osu.Game.Tests/Editing/Checks/CheckPreviewTimeTest.cs

I will change my PR to use the mentioned method(s) instead.